src/site/content/en/blog/prefers-color-scheme/index.md

```diff
@@ -464,6 +464,34 @@ in order to see the theme color and favicon changes, open the
   });
 ```
 
+## Debugging and testing dark mode
+
+### Emulating `prefers-color-scheme` in DevTools
+
+Switching the entire operating system's color scheme can get annoying real quick,
+so in Chrome DevTools, we now allow you to emulate the user's preferred color scheme
+in a way that only the currently visible tab is affected.
```

Comment:
Slightly simpler:

```suggestion
so Chrome DevTools now allows you to emulate the user's preferred color scheme
in a way that only affects the currently visible tab.
```

Reply:
maybe make "now allows you" a link to https://developers.google.com/web/updates/2019/10/devtools#userpreferences